tests/test_eks_get_image_type.py

```diff
@@ -0,0 +1,20 @@
+"""Test EKS Upgrade get image type specific logic."""
+from typing import Optional
+
+import pytest
+
+from eksupgrade.src.eks_get_image_type import image_type
+
+
+@pytest.mark.parametrize(
+    "node_type,image_id",
+    [
+        ("windows server 2019 datacenter ", "ami-ekswin"),
+        ("windows server 2022", "ami-ekswin"),
+        ("amazon linux 2", "ami-ekslinux"),
+    ],
+)
+def test_image_type(ec2_client, region, node_type, image_id) -> None:
+    """Test the loading_config method."""
```

Comment:
- (nit) - The docstring may need to be updated.

Reply:
Updated.  Thanks!